src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const errorMessage = 'Bzzz... Error!';
+  let count = 0;
+
+  const robot = (a) => {
+    count++;
+
+    return count > 3 && count % 2 === 0
+      ? (b) => {
+        return errorMessage;
+      }
+      : (b) => {
+        return a + b;
+      };
```

Comment:
Better to perform this in the inner function and define return value based on the condition.